tools/flaky-test-reporter/github_issue_test.go

```diff
@@ -105,15 +106,18 @@ func createRepoData(passed, flaky, failed, notenoughdata int, startTime int64) *
 func TestCreateIssue(t *testing.T) {
 	datas := []struct {
 		passed, flaky, failed, notenoughdata int
+		createissueflag											 bool
```

Comment:
```suggestion
		createissueflag bool
```

Reply:
ran gofmt, should be standard formatting now.